buildspecs/aix_ppc-64.spec

```diff
@@ -128,6 +128,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 		<flag id="build_j2se" value="true"/>
 		<flag id="build_java8" value="true"/>
 		<flag id="build_java9" value="false"/>
+		<flag id="build_jitserver" value="false"/>
```

Comment:
I added this flag in each spec in case we want to enable it later on other platforms.
Currently the openjdk build scripts rely on `<flag id="build_jitserver"` to be present to toggle it on when `--enable jitserver` is specified. See the changes in my PR https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/350.
In absence of these flags, I think it won't be very straight forward that we need to add `build_jitserver` flag in the spec file when enabling it for any new platform.

Reply:
This is the standard pattern we use for flags - they are, with some historical cruft aside, only added to the spec when when they are enabled.

Reply:
Are you suggesting to remove it from all specs because it not enabled by default in any. It gets set only if `--enable-jitserver` option is specified to configure step when building openjdk. Removing it from all specs would break ibmruntimes/openj9-openjdk-jdk8#350.

Reply:
Looking at some of the other features, it seems you can add the line to the core.feature file and not modify every spec file.

Reply:
yes, this would work.